sim/hunter/talents.go

```diff
@@ -116,11 +116,11 @@ func (hunter *Hunter) registerBestialWrathCD() {
 	}
 
 	actionID := core.ActionID{SpellID: 19574}
-
+	
 	bestialWrathPetAura := hunter.pet.RegisterAura(core.Aura{
 		Label:    "Bestial Wrath Pet",
 		ActionID: actionID,
-		Duration: time.Second * 18,
+		Duration: time.Second * core.TernaryDuration(core.HasItemEffect(MaelstromsWrath), 21, 18),
```

Comment:
This should be able to be done in the item effect definition by using a dummy aura with an `OnInit` callback. You would need to make this aura available through the hunter struct to do that